lib/elixir/lib/module.ex

```diff
@@ -1221,12 +1221,75 @@ defmodule Module do
     behaviours = :ets.lookup_element(table, :behaviour, 2)
     impls = :ets.lookup_element(table, {:elixir, :impls}, 2)
 
+    check_behaviours(env, behaviours, all_definitions)
+
     if impls != [] do
       non_implemented_callbacks = check_impls(behaviours, impls)
       warn_missing_impls(env, non_implemented_callbacks, all_definitions, overridable_pairs)
     end
   end
 
+  defp check_behaviours(env, behaviours, all_definitions) do
+    Enum.each(behaviours, fn behaviour ->
+      cond do
+        is_standard_behaviour(behaviour) ->
```

Comment:
Please call this function `standard_behaviour?/1` as the `is_` prefix is conventionally reserved for predicates that can be used in guards (like `is_boolean/1`).

Reply:
No problem.
Old habits die hard 😆